dll/win32/wtsapi32/wtsapi32.c

```diff
@@ -626,8 +629,21 @@ BOOL WINAPI WTSQueryUserConfigW(LPWSTR pServerName, LPWSTR pUserName, WTS_CONFIG
  */
 BOOL WINAPI WTSRegisterSessionNotification(HWND hWnd, DWORD dwFlags)
 {
-    FIXME("Stub %p 0x%08x\n", hWnd, dwFlags);
-    return TRUE;
+  DWORD processId;
+
+  if (IsWindow(hWnd) != 0) {
+    GetWindowThreadProcessId(hWnd, &processId);
+    if (processId != GetCurrentProcessId()) {
+        SetLastError(0x580);
```

Comment:
I still need to sus out the defined constant for this magic number back in Ghidra

Reply:
Just use MSDN.